<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenClusterDialog</class>
 <widget class="QDialog" name="GenClusterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>713</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>924</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>924</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cluster generating dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="2" colspan="4">
    <widget class="QTableWidget" name="InputTable">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="QListWidget" name="ClusterList">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="4">
    <widget class="QTextEdit" name="Information">
     <property name="minimumSize">
      <size>
       <width>294</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>294</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="LRows">
     <property name="text">
      <string>Rows</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QSpinBox" name="SB_Rows"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="LNumbers">
     <property name="text">
      <string>Numbers</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="PB_Add">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Add list</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="PB_Gen">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="PB_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QSpinBox" name="SB_Numbers"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="View3D">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
